tools/schemacode/bidsschematools/tests/conftest.py

```diff
@@ -80,14 +80,18 @@ def fixture():
 @pytest.fixture(scope="session")
 def schema_dir():
     """Path to the schema housed in the bids-specification repo."""
+    from bidsschematools import utils
```

Comment:
I think I did this to see if I could get away with not installing the test dependencies. I couldn't, so I can revert if you like.

Reply:
I think it's fine to import here :)